devicon.json

```
@@ -8945,5 +8945,27 @@
         },
         "color": "#F04E98",
         "aliases": []
+    },
+    {
+        "name": "p5js",
+        "altnames": [
+            "p5.js"
+        ],
+        "tags": [
+           "javascript",
+           "js",
+           "library"
+        ],
+        "versions": {
+            "svg": ["original-wordmark"],
+            "font": ["original-wordmark"]
+        },
+        "color": "#ED225D",
+        "aliases": [
+            {
+                "base": "original-wordmark",
+                "alias": "plain-wordmark"
+            }
+        ]
```

Since the wordmark version here works both as a non-wordmark and wordmark version, we should change the icon name to p5js-original.svg, and add aliases from `original` to `plain`, `original-wordmark` and `plain-wordmark`. 

Why do we use aliases, and why should the icon be named `original` instead of `plain`, `original-wordmark` or `plain-wordmark`? You can read more about it in our wiki post about [SVG-Versions](https://github.com/devicons/devicon/wiki/SVG-Versions#note) (see note 3. at the bottom)

```suggestion
        "versions": {
            "svg": ["original"],
            "font": ["original"]
        },
        "color": "#ED225D",
        "aliases": [
            {
                "base": "original",
                "alias": "plain"
            },
            {
                "base": "original",
                "alias": "original-wordmark"
            },
            {
                "base": "original",
                "alias": "plain-wordmark"
            }
        ]
```